__mocks__/kbar.ts

```diff
@@ -0,0 +1,7 @@
+const kbar: any = jest.createMockFromModule('kbar');
```

Comment:
the change to require kbar from breadcrumbs has given the tests a bit of bother. 
rather than mock up kbar properly, I'm just ignoring 😁  we aren't testing kbar and there aren't mocks available.. so.. 👋